Product/ProductService.js

```diff
@@ -0,0 +1,70 @@
+import axios from 'axios'
+
+
+let url = 'https://panda-market-api-crud.vercel.app/products'
+
+export async function getProductList(page=1, pageSize=10, keyword=""){
+    
+    try{
+        let response = await axios.get(url,
+            {params: {
+                page : page,
+                pageSize : pageSize,
+                keyword : keyword
+            }});
+        return response.data;
+    }catch(error){
+        console.error(error);
+    }
+}
+
+    
+
+export async function getProduct(productId){
+    url= `https://panda-market-api-crud.vercel.app/products/${productId}`
+    try{
+        await axios.get(url);
+    }catch(error){
+        console.error(error);
+    }
+}
+
+export async function createProduct({name="",description="",price=0, tags="",images=""}){
```

Comment:
변수명을 보면 tags와 images는 배열이 되어야 할 것 같아요. 만약 배열이어야 한다면 다음과 같이 작성 가능하겠죠?
```suggestion
export async function createProduct({name="",description="",price=0, tags=[],images=[]}){
```